source/2019-12-07-countdown-to-the-new-year-ember-animated.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Countdown to The New Year: Ember Animated
+author: Jen Weber
+tags: 2019, Addons, DecEmber
+alias: 2019-12-07-countdown-to-the-new-year-ember-animated.md
+responsive: true
+---
+
+<b>This is the seventh in our DecEmber series- <i>"Countdown to The New Year: 31 Days of Ember Addons"</i>.  We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!</b>
```

Comment:
```suggestion
**This is the seventh in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
```

bringing in edits from https://github.com/ember-learn/ember-blog/pull/414 ... some regular markdown stuff like `<i>` doesn't work in this repo